src/Components/Calculator.js

```diff
@@ -1,121 +1,89 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      element: {
-        total: null,
-        next: null,
-        operation: null,
-      },
-    };
-    this.result = this.result.bind(this);
-  }
+const Calculator = () => {
+  const [element, AddNewEl] = useState({
```

Comment:
[OPTIONAL] Following JavaScript best practices, the state method should be written in camelCase. It would be nicer to change AddNewEl to `addNewElement`.